### PR TITLE
Adds a tag for bootstrap modals 

### PR DESCRIPTION
**Pull request by meg1502**
Instead of three seperate tags as in wondergem, the whole modal can now be constructed using the t:modal tag

### CODE REVIEW

File: src/main/resources/default/assets/tycho/styles/misc.scss [hunk 1]
```diff
@@ -49,3 +49,7 @@ a {
 .back-button {
   display: none;
 }
+
+.hidden {
+  display: none !important;
+}
```
Comment (sabieber) [inline src/main/resources/default/assets/tycho/styles/misc.scss:55]: Das gibts glaub schon als `.d-none` usw: https://getbootstrap.com/docs/5.0/utilities/display/#hiding-elements

**Review by sabieber — CHANGES_REQUESTED** [review]

File: src/main/resources/default/assets/tycho/styles/typeface.scss [hunk 2]
```diff
@@ -7,9 +7,49 @@
...
 .text-black-75 {
-  color: rgba(0,0,0,0.75) !important;
+  color: rgba(0, 0, 0, 0.75) !important;
 }
 
 .display-5 {
   font-size: 2.5rem;
 }
+
+.text-lg11 {
```
Comment (andyHa) [inline src/main/resources/default/assets/tycho/styles/typeface.scss:17]: do we need all of them? oder s.th. like "small" and "smaller" or the like?

File: src/main/resources/default/taglib/t/modal.html.pasta [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<i:arg name="name" type="String"/>
+<i:arg name="titleKey" type="String" default=""/>
+<i:arg name="title" type="String" default="@i18n(titleKey)"/>
+<i:arg type="String" name="submitKey" default=""/>
```
Comment (andyHa) [inline src/main/resources/default/taglib/t/modal.html.pasta:4]: optional: also add submit etc. to permit to supply non-i18n-ed values(?)

File: src/main/resources/default/taglib/t/modal.html.pasta [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+<div id="@name" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="@(name + 'Title')">
+    <div class="modal-dialog @class" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <i:if test="isFilled(title)">
+                    <h4 class="modal-title" id="@(name + 'Title')">@title</h4>
+                </i:if>
+                <button type="button" class="close" data-dismiss="modal" aria-label="@i18n('NLS.close')">
+                    <span aria-hidden="true">&times;</span>
```
Comment (andyHa) [inline src/main/resources/default/taglib/t/modal.html.pasta:21]: optional: fa icon(?)

**Review by sabieber — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

**Pull request merged**